packages/components/card/src/card.tsx

```diff
@@ -25,12 +27,34 @@ export type TCardProps = {
    */
   className?: string;
   children?: ReactNode;
+  /**
+   * The callback function to be executed when the Card component is clicked. Prefer this for managing side effects rather than navigation.
+   */
+  onClick?: () => void;
+  /**
+   * The URL that the Card should point to. If provided, the Card will be rendered as an anchor element.
+   */
+  to?: string | LocationDescriptor;
+  /**
+   * A flag to indicate if the Card points to an external source.
+   */
+  isExternalLink?: boolean;
+  /**
+   * Indicates that a clickable Card should not allow clicks. This allows consumers to temporarily disable a clickable Card.
+   */
+  isDisabled?: boolean;
 };
 
-const Card = (props: TCardProps) => (
-  <div
-    {...filterDataAttributes(props)}
-    css={css`
+const Card = (props: TCardProps) => {
+  const isClickable = Boolean(!props.isDisabled && (props.onClick || props.to));
+  // Only disable styling if the card is not clickable
+  const isDisableEnabled = props.isDisabled && (props.onClick || props.to);
```

Comment:
Maybe a tricky to understand name 🤣 

What about `shouldBeDisabled`?

Reply:
Hahaha, yeah...not my finest work 😅 

Addressed in 4c8f203565a1e0d9e46c4589b6306a575ee64049